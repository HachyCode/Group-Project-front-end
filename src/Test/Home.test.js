import React from 'react';
import ReactDOM from 'react-dom';
import TopBoxTextContainer from './../Pages/Home';
import {cleanup} from '@testing-library/react';
import renderer from 'react-test-renderer'

afterEach(cleanup);
test("renders Home page", () => {
	const div = document.createElement("div");
	ReactDOM.render(
		<TopBoxTextContainer>
			"Wear a mask, so you could save the world"
			-Batman
		</TopBoxTextContainer> , div
	)
})

test("mathches snapshot", () => {
	const tree = renderer.create(
		<TopBoxTextContainer>
			"Wear a mask, so you could save the world"
			-Batman
		</TopBoxTextContainer>
	).toJSON();
	expect(tree).toMatchSnapshot();
})