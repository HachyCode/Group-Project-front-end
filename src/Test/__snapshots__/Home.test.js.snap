// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mathches snapshot 1`] = `
<div
  className="sc-crrsfI joKIVN"
>
  <div
    className="sc-dQppl fEytmg"
  >
    <img
      alt="logo"
      className="sc-bqyKva gdekPE"
      src="black_logo.png"
    />
    <div
      className="sc-kstrdz eMSxFA"
    >
      "Wear a mask, so you could save the world" -Batman
    </div>
  </div>
  <div
    className="sc-bdfBwQ iOgsXs sc-kEjbxe hvfeZo"
    onClick={[Function]}
  >
    <div
      className="sc-gsTCUz gWTlLh"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-exclamation-triangle fa-w-18 sc-dlfnbm iTcowd"
        data-icon="exclamation-triangle"
        data-prefix="fas"
        focusable="false"
        newIcon={[Function]}
        role="img"
        style={Object {}}
        viewBox="0 0 576 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <img
      alt="supplierSelectino"
      className="sc-hKgILt kMEHpc"
      src="notification.jpg"
    />
    <div
      className="sc-eCssSg bdoMYU"
    >
      <h3
        className="sc-jSgupP eplsmV"
      >
        Supplier selection needed
      </h3>
      <p
        className="sc-gKsewC dycbFE"
      >
        You need to specify the supplier on PoForm with ID:_________
      </p>
    </div>
    <div
      className="sc-iBPRYJ eYhFbe"
    >
      <p
        className="sc-fubCfw ktlUso"
      >
        07/12/2022
      </p>
    </div>
    <button
      className="sc-pFZIQ kpgjur"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times-circle fa-w-16 sc-jrAGrp iPcKdq"
        data-icon="times-circle"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </div>
  <div
    className="sc-bdfBwQ iOgsXs sc-kEjbxe hvfeZo"
    onClick={[Function]}
  >
    <div
      className="sc-gsTCUz gWTlLh"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-exclamation-triangle fa-w-18 sc-dlfnbm iTcowd"
        data-icon="exclamation-triangle"
        data-prefix="fas"
        focusable="false"
        newIcon={[Function]}
        role="img"
        style={Object {}}
        viewBox="0 0 576 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <img
      alt="itemQuantity"
      className="sc-hKgILt kMEHpc"
      src="notification.jpg"
    />
    <div
      className="sc-eCssSg bdoMYU"
    >
      <h3
        className="sc-jSgupP eplsmV"
      >
        Item quantity required
      </h3>
      <p
        className="sc-gKsewC dycbFE"
      >
        PoForm ID:_________ : Item with ID:_________ requires a quantity
      </p>
    </div>
    <div
      className="sc-iBPRYJ eYhFbe"
    >
      <p
        className="sc-fubCfw ktlUso"
      >
        07/12/2022
      </p>
    </div>
    <button
      className="sc-pFZIQ kpgjur"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times-circle fa-w-16 sc-jrAGrp iPcKdq"
        data-icon="times-circle"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </div>
  <div
    className="sc-bdfBwQ cMVoHA sc-kEjbxe hvfeZo"
    onClick={[Function]}
  >
    <div
      className="sc-gsTCUz gWTlLh"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-exclamation-triangle fa-w-18 sc-dlfnbm gOGoyI"
        data-icon="exclamation-triangle"
        data-prefix="fas"
        focusable="false"
        newIcon={[Function]}
        role="img"
        style={Object {}}
        viewBox="0 0 576 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.054c-36.937 0-59.999-40.055-41.577-71.987L246.423 23.985c18.467-32.009 64.72-31.951 83.154 0l239.94 416.028zM288 354c-25.405 0-46 20.595-46 46s20.595 46 46 46 46-20.595 46-46-20.595-46-46-46zm-43.673-165.346l7.418 136c.347 6.364 5.609 11.346 11.982 11.346h48.546c6.373 0 11.635-4.982 11.982-11.346l7.418-136c.375-6.874-5.098-12.654-11.982-12.654h-63.383c-6.884 0-12.356 5.78-11.981 12.654z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </div>
    <img
      alt="LowSupplies"
      className="sc-hKgILt kMEHpc"
      src="notification.jpg"
    />
    <div
      className="sc-eCssSg bdoMYU"
    >
      <h3
        className="sc-jSgupP eplsmV"
      >
        Item low on stock
      </h3>
      <p
        className="sc-gKsewC dycbFE"
      >
        Item with ID:_________ is running low on stock
      </p>
    </div>
    <div
      className="sc-iBPRYJ eYhFbe"
    >
      <p
        className="sc-fubCfw ktlUso"
      >
        07/12/2022
      </p>
    </div>
    <button
      className="sc-pFZIQ kpgjur"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-times-circle fa-w-16 sc-jrAGrp iPcKdq"
        data-icon="times-circle"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm121.6 313.1c4.7 4.7 4.7 12.3 0 17L338 377.6c-4.7 4.7-12.3 4.7-17 0L256 312l-65.1 65.6c-4.7 4.7-12.3 4.7-17 0L134.4 338c-4.7-4.7-4.7-12.3 0-17l65.6-65-65.6-65.1c-4.7-4.7-4.7-12.3 0-17l39.6-39.6c4.7-4.7 12.3-4.7 17 0l65 65.7 65.1-65.6c4.7-4.7 12.3-4.7 17 0l39.6 39.6c4.7 4.7 4.7 12.3 0 17L312 256l65.6 65.1z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </button>
  </div>
</div>
`;
